docs/HeliumParamSpecs.md

```diff
@@ -0,0 +1,295 @@
+# Helium Parameter Specs Documentation
+
+## System Overview
+
+Helium is WebAPI reference application designed to "fork and code" in c#, java, typescript languages with the following features:
+
+* Securely build, deploy and run an App Service (Web App for Containers) application
+* Use Managed Identity to securely access resources
+* Securely store secrets in Key Vault
+* Securely build and deploy the Docker container from Container Registry or Azure DevOps
+* Connect to and query CosmosDB
+* Automatically send telemetry and logs to Azure Monitor
+* Instructions for setting up Key Vault, ACR, Azure Monitor and Cosmos DB are in the Helium language specific repo readme.
+
+## Design Considerations
+
+## Assumptions and Dependencies
+
+### General Constraints
+
+### Goals and Guidelines
+
+### Development Methods
+
```

Comment:
I think we can remove the unused headers to make it quicker to get to the spec definition/content.

Reply:
Removed